### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by RodrigoGarciaLab**
#### What does this PR do?
Creates the dashboard view with the graphics (with hardcoded data)
...

#### Where should the reviewer start?
dashboard.css
...

#### How should this be manually tested?

...

#### Any background context you want to provide?

...

#### What are the relevant issues?

...

#### Screenshots (if appropriate)
screen shot 2018-10-12 at 2 51 31 pm

...

#### Checklist:

* [ ] If you've fixed a bug or added code that should be tested, add tests!

---

#### Database changes
None
...

#### Deployment instructions (Specific order, ENV variables, etc...)

...

### CODE REVIEW

**Comment by nbassagoda** [comment]
Ignorar codeclimate porque son todos problemas en el codigo harcodeado del controller que se va a ir todo despues

File: app/assets/stylesheets/side-nav.css [hunk 1]
```diff
@@ -53,6 +53,7 @@
 
 .mdl-layout__drawer {
   background: #272727 !important;
+  width: 20% !important;
```
Comment (mconiglio) [inline app/assets/stylesheets/side-nav.css]: set a specific width so the navbar size doesn't depend on the screen size. according to the design this should be around 290px

File: app/assets/stylesheets/timer-table.css [hunk 2]
```diff
@@ -3,3 +3,7 @@
   width: 100%;
   height: 100%;
 }
+
+.header {
+  margin-left: 20% !important;
```
Comment (mconiglio) [inline app/assets/stylesheets/timer-table.css]: same. also use media queries for this to depend on screen size

File: db/schema.rb [hunk 3]
```diff
@@ -16,7 +16,7 @@
   enable_extension "plpgsql"
 
   create_table "project_users", force: :cascade do |t|
-    t.boolean "project_manager", default: false, null: false
+    t.boolean "project_manager", default: false
```
Comment (mconiglio) [inline db/schema.rb]: why did you remove the `null` option in these lines?

File: app/controllers/dashboard_controller.rb [hunk 4]
```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+# rubocop:disable all
```
Comment (mconiglio) [inline app/controllers/dashboard_controller.rb]: why did you add this?

Comment (RodrigoGarciaLab) [reply app/controllers/dashboard_controller.rb]: We wrote some dummy data to create de charts just so the next one who works with the dashboard knows how to use it. This dummy data is generating many rubocop alerts so we deactivated it since that dummy data won't exist actually.
